### ISSUE TITLE
Impact on rewards of extended witness accounts

### ISSUE THREAD

**Issue by RezDaBombDiggy** [Gulden/gulden-official#211]
### Describe the issue
When extending two witness accounts I noticed that in the following 48 hours NONE of my active witness accounts (six in total, using Pi with G-Dash for witnessing) received any rewards, whereas I normally received 10-15 rewards a day. Have been patient as anomalies may occur but decided to report after 48+ hours.

### Can you reliably reproduce the issue?
No, though description above might suffice. Steps taken listed below.

#### If so, please list the steps to reproduce below:
1. Rebooted Pi (to no avail) 
2. Left desktop wallet open for 12 hours after rescanning transactions and rewards slowly started coming in after several hours
3. Closed desktop wallet leaving only Pi to witness. Received 4 rewards on different accounts from the ones extended on that day.
4. Received first reward on extended account after ~60 hours of extending.

### Expected behaviour
Stating the obvious here; continue witnessing as per normal with similar levels of rewards.

### Actual behaviour
See above.

### Screenshots.
N/A

### What version of Gulden are you using?
Desktop wallet (win 64-bit) 2.2.0.7
G-Dash 1.3 (if relevant at all)

### Machine specs:
N/A - can be provided upon request

### Any extra information that might be useful in the debugging process.
Let me know if needed

**Comment by mjmacleod** [comment]
Witness accounts perform according to their weight, there are no known bugs in this regard and as this is incredibly well tested there are unlikely to be any.

The most likely explanation is one of either:
1) Your devices are out of sync
2) Your account weights have dropped

2 seems the most likely, to investigate further I would need to know your account weights before and after you extended them, or alternatively the witness addresses in question so that I can look this up myself.

Its possible that (1) has also occured as a result of rebooting your pi etc. - so please check that it is properly synced with the chain to be sure

**Comment by RezDaBombDiggy** [comment]
All was (and is) in sync as far as I coudl verify this through G-Dash dashboard as well as the desktop wallet. I have verified this multiple times before raising the issue. 

Quite happy to provide addresses but prefer not to do this through a public forum, will contact you through Slack.

Also wanted to raise another issue I encountered; the extended witness accounts started showing their old (pre-Phase 4) address in the desktop wallet. Have kept a log of old and new addresses / keys through which I found out about this. Could be a separate issue but wanted to let you know anyway.
